packages/spark-core/spark-core-prerender.js

```diff
@@ -1,9 +1,7 @@
 import detectJS from './utilities/detectJS';
-import typeLoader from './base/typography';
 
-const SparkCorePrerender = ({ typeConfig = {} } = {}) => {
+const SparkCorePrerender = () => {
```

Comment:
I think the documentation needs to be updated. I think we show the typeConfig being passed in our example.